rust/lance-file/src/v2/reader.rs

```diff
@@ -57,6 +57,18 @@ pub struct BufferDescriptor {
     pub size: u64,
 }
 
+#[derive(Debug)]
+pub struct FileStatistics {
+    pub columns: Vec<ColumnStatistics>,
+}
+
+// Currently we report the `num_pages` and `size_bytes` of each column in Lance file.
+#[derive(Debug)]
+pub struct ColumnStatistics {
+    pub num_pages: usize,
+    pub size_bytes: usize,
```

Comment:
Given 100GB files are possible, we probably should make `size_bytes` 64-bits on all platforms.
```suggestion
    pub num_pages: u64,
    pub size_bytes: u64,
```

Reply:
yeah, u64 is better here, thanks for the advise.

Reply:
changed to use `u64` to represent `size_bytes`, `num_pages` still uses `usize`.